### PR TITLE
dump: Add to print 'info' bit name

### PR DESCRIPTION
**Pull request by daeroro**
When executing 'dump' command, uftrace prints 'file header'.
And it prints 'feature mask' and its bit name.
(e.g. "PLTHOOK", "KERNEL", ..)

Similarly, it prints 'info mask' value. So, I added simple logic
to print 'info' bit name too.(e.g. "EXE_NAME", "CMDLINE", ...)

Signed-off-by: DaeRo Lee <skseofh@naver.com>

### CODE REVIEW

File: cmds/dump.c [hunk 1]
```diff
@@ -544,10 +548,13 @@ static void dump_raw_header(struct uftrace_dump_ops *ops, struct uftrace_data *h
 	       handle->hdr.endian == 1 ? "little" : "big");
 	pr_out("uftrace file header: class         = %u (%s bit)\n", handle->hdr.elf_class,
 	       handle->hdr.elf_class == 2 ? "64" : "32");
-	get_feature_string(buf, sizeof(buf), handle->hdr.feat_mask);
+	get_header_string(buf, sizeof(buf), handle->hdr.feat_mask, feat_str, FEAT_BIT_MAX);
 	pr_out("uftrace file header: features      = %#" PRIx64 " (%s)\n", handle->hdr.feat_mask,
 	       buf);
-	pr_out("uftrace file header: info          = %#" PRIx64 "\n", handle->hdr.info_mask);
+	memset(buf, 0, sizeof(buf));
```
Comment (honggyukim) [inline cmds/dump.c:554]: Do we need this `memset`?

**Review by honggyukim — COMMENTED** [review]
Thanks for your patch! But I think we have to clarify #1444 first before this change. @namhyung Please have a look.

File: cmds/dump.c (hunk 1 above)
Comment (daeroro) [reply cmds/dump.c:554]: I thought that `buf` needs to be cleared if feature name length is shorter than info name length.
But `snprintf` includes the terminating null byte ('\0')) to `buf`, so `memset` doesn't need.
Thanks for your comment :)

**Comment by namhyung** [comment]
Right, it'd be nice to include cleanup in #1444.

File: cmds/dump.c [hunk 2]
```diff
@@ -533,6 +530,13 @@ static void dump_raw_header(struct uftrace_dump_ops *ops, struct uftrace_data *h
 	int i;
 	char buf[1024];
 	struct uftrace_raw_dump *raw = container_of(ops, typeof(*raw), ops);
+	const char *feat_str[] = { "PLTHOOK",	 "TASK_SESSION", "KERNEL",     "ARGUMENT",
+				   "RETVAL",	 "SYM_REL_ADDR", "MAX_STACK",  "EVENT",
+				   "PERF_EVENT", "AUTO_ARGS",	 "DEBUG_INFO", "ESTIMATE_RETURN" };
+	const char *info_str[] = { "EXE_NAME",	   "EXE_BUILD_ID", "EXIT_STATUS", "CMDLINE",
+				   "CPUINFO",	   "MEMINFO",	   "OSINFO",	  "TASKINFO",
+				   "USAGEINFO",	   "LOADINFO",	   "ARG_SPEC",	  "RECORD_DATE",
+				   "PATTERN_TYPE", "VERSION" };
```
Comment (namhyung) [inline cmds/dump.c:539]: While it was not there originally, please put a comma after the last element so that it can format the code 'properly'.

**Review by namhyung — APPROVED** [review]
LGTM

**Pull request merged**